amd/src/quizgame.js

```diff
@@ -588,10 +595,19 @@ define(['jquery','core/yui', 'core/notification', 'core/ajax'], function($, Y, n
         wrapText(context, this.text, true, 17, displayRect.width * 0.2, this.x + this.image.width / 2, this.y - 5);
     };
     Enemy.prototype.die = function() {
+        this.die(true);
+    };
+    Enemy.prototype.die = function(increaseScore) {
```

Comment:
What was the need for the increaseScore? Won't it just always call this with true?

Reply:
No, it's called on line 661 when a MatchEnemy dies. This is used because with a MatchEnemy there are two ships, and if one is killed for half the points and then the other is killed for the other half, rounding issues can occur (e.g. It looking like the score jumps an extra point or two, or the score is a number or two off due to rounding each one). The correct score amount is then added manually right afterward on line 679.